website_base_multi_image/static/src/js/website_multi_image.js

```diff
@@ -0,0 +1,96 @@
+/* © 2016 Tecnativa, S.L.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+ */
+odoo.define("website_base_multi_image.images",
+function(require){
+"use strict";
+
+var animation = require("web_editor.snippets.animation");
+var core = require("web.core");
+var Model = require("web.Model");
+var Session = require("web.Session");
+var _t = core._t;
+var $ = require("$");
+
+animation.registry.website_base_multi_image =
+animation.Class.extend({
+    selector: ".js_image_container",
+    start: function (editable_mode) {
+        if (editable_mode) {
+            return;
+        };
+        this.$image_preview_thumb = this.$(".thumbnail-images");
+        this.$main_image = this.$("#main_image");
+        this.$file_input = this.$("#file_input");
+        this.$label_input = this.$(".image_upload");
+        this.$file_input_multi = this.$("#file_input_multi");
```

Comment:
If there are chances this widget can appear more than once in a webpage, it's better to use classes than ids, because it is expected that there's only 1 single element with a given id in a page.